packages/protoplugin/package.json

```diff
@@ -13,17 +13,30 @@
     "clean": "rm -rf ./dist/cjs/* ./dist/esm/* ./dist/types/*",
     "build": "npm run build:cjs && npm run build:esm+types",
     "build:cjs": "../../node_modules/typescript/bin/tsc --project tsconfig.json --module commonjs --outDir ./dist/cjs && echo >./dist/cjs/package.json '{\"type\":\"commonjs\"}'",
-    "build:esm+types": "../../node_modules/typescript/bin/tsc --project tsconfig.json --module ES2015 --outDir ./dist/esm --declaration --declarationDir ./dist/types"
+    "build:esm+types": "../../node_modules/typescript/bin/tsc --project tsconfig.json --module ES2015 --outDir ./dist/esm --declaration --declarationDir ./dist/types && cp ./dist/types/index.d.ts ./dist/types/index.d.cts && cp ./dist/types/ecmascript/index.d.ts ./dist/types/ecmascript/index.d.cts",
```

Comment:
That's not very maintainable :sweat_smile: 

When I looked into this previously, I noticed that it is actually pretty cheap to emit types in addition to js. So I think it should be possible to simply emit types when building cjs _and_ esm, and have a .d.ts file next to each .js file.

Reply:
Updated `packages/protobuf` accordingly (I think). Is that what you mean? If so, I can do the same to `protoplugin`.